packages/SidePanel/package.json

```diff
@@ -16,7 +16,7 @@
     "@babel/runtime-corejs2": "^7.3.1",
     "@paprika/button": "^0.2.3",
     "@paprika/heading": "^0.3.1",
-    "@paprika/helpers": "^0.2.4",
+    "@paprika/helpers": "^0.2.5",
```

Comment:
👍

Reply:
But then now side panel points to a "future version" and npx lerna bootstrap fails
`error Couldn't find any versions for "@paprika/helpers" that matches "^0.2.5"`

Reply:
I reverted for now, since its breaking. Let me know if thats okay.